stytch/config/config.go

```diff
@@ -1,6 +1,6 @@
 package config
 
-const APIVersion = "5.14.1"
+const APIVersion = "5.14.2"
```

Comment:
Your call as author, but I'd say this new functionality (new options in request params) is worth a minor version bump.
```suggestion
const APIVersion = "5.15.0"
```